c-api/object.po

```diff
@@ -6,59 +6,79 @@ msgstr ""
 "Project-Id-Version: Python 3.6\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-06-28 15:29+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2018-10-17 19:20+0200\n"
 "Language-Team: FRENCH <traductions@lists.afpy.org>\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: \n"
+"X-Generator: Poedit 2.2\n"
 
 #: ../Doc/c-api/object.rst:6
 msgid "Object Protocol"
-msgstr ""
+msgstr "Protocole Objet"
 
 #: ../Doc/c-api/object.rst:11
 msgid ""
 "The ``NotImplemented`` singleton, used to signal that an operation is not "
 "implemented for the given type combination."
 msgstr ""
+"Le singleton ``NotImplemented``, utilisé pour signaler qu'une opération pour "
+"la combinaison de type en question."
 
 #: ../Doc/c-api/object.rst:17
 msgid ""
 "Properly handle returning :c:data:`Py_NotImplemented` from within a C "
 "function (that is, increment the reference count of NotImplemented and "
 "return it)."
 msgstr ""
+"Traite proprement le renvoi de :c:data:`Py_NotImplemented` depuis "
+"l'intérieur d'une fonction C (c'est-à-dire, incrémente le compteur de "
+"référence de NotImplemented et le retourne)."
 
 #: ../Doc/c-api/object.rst:24
+#, fuzzy
 msgid ""
 "Print an object *o*, on file *fp*.  Returns ``-1`` on error.  The flags "
 "argument is used to enable certain printing options.  The only option "
 "currently supported is :const:`Py_PRINT_RAW`; if given, the :func:`str` of "
 "the object is written instead of the :func:`repr`."
 msgstr ""
+"Affiche un objet *o*, sur le fichier *fp*. Retourne \"-1\" en cas d'erreur. "
+"L'argument flags est utilisé pour permettre certaines options d'affichage. "
+"La seule option actuellement supportée est :const:`Py_PRINT_RAW`; si donné, "
```

Comment:
Nous préférons "gérer" (ou autres) pour "to support"  plutôt que "supporter" qui est un anglicisme.